### DISCUSSION TITLE
JBot & gammabot

### DISCUSSION THREAD

**Discussion by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#177] [Show and tell]
A little JBot and gammabot context..

# JBot
* currently our IRC bot we mostly love
* handles the live show title suggestions at [JBTitles.com](https://jbtitles.com/)
* code: https://github.com/JupiterBroadcasting/jbot-showbot

# gammabot
* currently JB's Matrix bot we totally love
* is the natural successor to JBot
* code: https://github.com/JupiterBroadcasting/gammabot

**Comment by elreydetoda** [comment]
Awesome! Wherever I get a chance I can't wait to look at it! Looks like gammabot is using python, from a quick glance 😁

**Reply by gerbrent** [reply]
_Queue Python Jingle!_

; )
